sql/database.sql

```diff
@@ -2662,9 +2662,9 @@ DROP TABLE IF EXISTS `lang_custom`;
 CREATE TABLE `lang_custom` (
   `lang_description` varchar(100) NOT NULL default '',
   `lang_code` char(2) NOT NULL default '',
-  `constant_name` mediumtext NOT NULL default '',
-  `definition` mediumtext NOT NULL default ''
-) ENGINE=MyISAM ;
+  `constant_name` mediumtext NOT NULL,
+  `definition` mediumtext
```

Comment:
Hi Oleg,
So, to clarify, both "mediumtext" and "mediumtext NOT NULL" are ok in INNODB.
And then to clarify:
1. "mediumtext" will set to NULL if nothing is given?
2. "mediumtext NOT NULL" will set to ''(ie. blank) if nothing is given?